### PR TITLE
Port CircleCI workflow to GitHub Actions

### PR DESCRIPTION
**Pull request by leonm1**
GitHub actions allows us unlimited runs per month and 20 concurrent
actions. This PR is partly designed to test which one is faster.

### CODE REVIEW

**Comment by leonm1** [comment]
GitHub Actions: 2m37s
CircleCI: 3m10s

21% speedup!

**Review by cktang88 — APPROVED** [review]
lgtm

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      - run: npm run build
+  audit:
+    name: Audit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - run: npm audit --audit-level=high --only=prod
```
Comment (cktang88) [inline .github/workflows/ci.yml:30]: imo audit doesn't need to be run on every PR, should be run local every once in a while

Comment (cktang88) [reply .github/workflows/ci.yml:30]: or author of PR should post results of running it locally if they update/add deps

Comment (leonm1) [reply .github/workflows/ci.yml:30]: What about running it on push to main? It wouldn't block PRs, would be automatic, and would be good info to know

**Pull request merged**